src/sentry/static/sentry/app/components/autoComplete.jsx

```diff
@@ -29,13 +29,19 @@ class AutoComplete extends React.Component {
      * Currently, this does not act as a "controlled" prop, only for initial state of dropdown
      */
     isOpen: PropTypes.bool,
+    /**
+     * If input should be considered an "actor". If there is another parent actor, then this should be `false`.
+     * e.g. You have a button that opens this <AutoComplete> in a dropdown.
+     */
+    inputIsActor: PropTypes.bool,
```

Comment:
i'm confused by the word actor here, because of the overlap with a sentry actor. is that your own terminology, and if so, could you call it something like hasParent or isHosted? (preferably something better)

Reply:
Yeah "actor" refers to the component that controls the visibility of the dropdown. It's used throughout `dropdownMenu.jsx`.